tests/tck/features/schema/Comment.feature

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Schema Comment
+
+  Scenario: Space Comment
+    Given an empty graph
+    When executing query:
+      """
+      CREATE SPACE test_comment(comment = 'This is a comment of space.');
+      """
+    Then the execution should be successful
+    When try to execute query:
+      """
+      SHOW CREATE SPACE test_comment;
+      """
+    Then the result should be, in any order:
+      | Space          | Create Space                                                                                                                                                                                                     |
+      | "test_comment" | "CREATE SPACE `test_comment` (partition_num = 100, replica_factor = 1, charset = utf8, collate = utf8_bin, vid_type = FIXED_STRING(8), atomic_edge = false, comment = 'This is a comment of space.') ON default" |
```

Comment:
Prefer to place comment outside the space attributes, format like:

```
CRAETE SPACE nba(partition_num=3) ON default, COMMENT [=] "nba space comment"
```

you can reference the mysql `create table` [documents.](https://dev.mysql.com/doc/refman/8.0/en/create-table.html)

Reply:
create table is same as our tag/edge syntax. For space better reuse the space option syntax.

Reply:
Not agree with you. We should unify the syntax for user. I don't think there's any difference between space and tag for comment。

Reply:
what's important, In my opinion, is that comment is not an option.

Reply:
Make sense, fixed.